transport/messages/result_message_base.hh

```diff
@@ -41,6 +43,19 @@ public:
         _custom_payload[key] = value;
     }
 
+    void add_tablet_info(locator::tablet_replica_set tablet_replicas, std::pair<dht::token, dht::token> token_range) {
+        if (!tablet_replicas.empty() && token_range.first.data().size() == token_range.second.data().size() && token_range.first.data().size() == 8) {
+            auto replicas_values = make_list_value(replica::replica_set_type, replica::replicas_to_data_value(tablet_replicas));
+            this->add_custom_payload("tablet_replicas", replicas_values.serialize_nonnull());
+            auto token_bytes = bytes();
+            auto first_token = token_range.first;
+            auto last_token = token_range.second;
+            token_bytes.append(first_token.data().data(), first_token.data().size());
```

Comment:
Now that I think about it, it's not safe to send in-memory representation. One reason is that there could be differences of byte ordering between machines (little vs big endian).

Compatible way would be:
```c++
auto v1 = data_value(dht::token::to_int64(first_token));
auto v2 = data_value(dht::token::to_int64(last_token));
bytes b(v1.serialized_size() + v2.serialized_size());
auto i = b.begin();
v1.serialize(i);
v2.serialize(i);
```

Then you don't need the checks for sizes because cql type values must be compatible.

The CQL type used here is the same as that of "last_token" column in the system.tablets schema.